packages/utilities/moment-timezone-include/README.md

```diff
@@ -0,0 +1,37 @@
+# @bbc/moment-timezone-include - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Futilities%2Fmoment-timezone-include%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Futilities%2Fmoment-timezone-include%2Fpackage.json) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/moment-timezone-include.svg)](https://www.npmjs.com/package/@bbc/moment-timezone-include) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
+
+This package allows you to exclude all moment timezones from being included, while including specific ones in the chunks you desire.
+
+## Installation
+
+```jsx
+npm install @bbc/moment-timezone-include --save
+```
+
+## Usage - Exclude default timezones
+
+```jsx
+plugins: [new MomentTimezoneInclude({ startYear: 1990, endYear: 2025 })],
+```
+
+This does two things. Firstly it removes all timezones from the moment-timezone package so that they can be included individually. It also specifies the date range of the specifically included timezone data. If `startYear` isnt provided it defaults to the earliest availile data and likewise if `endYear` isnt included it includes all known future data.
```

Comment:
```suggestion
This does two things. Firstly it removes all timezones from the moment-timezone package so that they can be included individually. It also specifies the date range of the specifically included timezone data. If `startYear` isn't provided it defaults to the earliest available data and likewise if `endYear` isn't included it includes all known future data.
```

Reply:
Also, do you mean "date" when you say "data" here?

Reply:
Nope, data :)